.eslintrc.js

```diff
@@ -1,21 +1,84 @@
 module.exports = {
-  extends: "eslint:recommended",
+ 'extends': [
+   'eslint-config-airbnb-base',
+   'eslint-config-airbnb-base/rules/strict',
+ ],
   parserOptions: {
     ecmaVersion: 6,
-    sourceType: "module",
+    sourceType: 'module',
     ecmaFeatures: {
       impliedStrict: true
     }
   },
+ settings: {
+     'import/resolver': {
+         'webpack': {}
+     }
+ },
   env: {
     browser: true,
     es6: true,
     amd: true,
     commonjs: true
   },
   rules: {
-    // Stylistic rules
-    "eol-last": "error",
-    "no-trailing-spaces": "error",
+    // TODO reactivate all these rules
+
+    // The following rule should be no more restrictive than
+    //'no-plusplus': ['error', { 'allowForLoopAfterthoughts': true }],
+    // to allow for i++ in for loops (even though we should mostly use for..of
+    // anyway)
+    'no-plusplus': 'off',
+
+    'func-names': 'off',
+    'no-mixed-operators': 'off',
+    'no-use-before-define': 'off',
+    'no-underscore-dangle': 'off',
+    'eqeqeq': 'off',
+    // what len ? Airbnb does 100. github wraps line above 80
+    'max-len': 'off',
+    'no-param-reassign': 'off',
+    'no-else-return': 'off',
+    'prefer-const': 'off',
+    'one-var': 'off',
```

Comment:
So I'm not even sure which one of `never` (several var, let, const) or `always` (only one `var`, `let` and `const` in a scope) we want to choose...